README.md

```diff
@@ -31,6 +31,20 @@ Dependency Parsers](https://www.aclweb.org/anthology/P19-1012) (see [acl2019 bra
 
 ## Usage
 
+### Install virtual env
+
+```sh
+python3.7 -m venv .venv
+source .venv/bin/activate
+python setup.py install
+```
+
+### Download training data and serialized model
+
+```sh
+dvc pull
```

Comment:
`dvc pull` => `ERROR: failed to pull data from the cloud - config file error: no remote specified. Create a default remote with
    dvc remote add -d <remote name> <remote url>`

Reply:
Wo liegen die Daten? Sind die Daten "public"? Wenn ein Password oder Token benötigt muss in der README.md stehen wer die Kontaktperson ist.

Reply:
Welche Daten meinst du genau? Der Parser ist im allgmeinen unabhängig von den Daten - müssen nur im CoNLL Format vorliegen. Für die aktuellen Modelle haben wir die Hamburger Treebank genutzt: https://corpora.uni-hamburg.de/hzsk/de/islandora/object/treebank:hdt

Reply:
Warum gibt es den Ordner `.dvc`? Zum Beispiel in https://github.com/zentrum-lexikographie/IMSnPars/tree/master/data/hdt sind lauter DVC Referenzen auf conllu Dateien, die eigentlich auf einem Remote Server herumliegen müssten